target/linux/kirkwood/patches-4.4/194-nsa325.patch

```diff
@@ -0,0 +1,257 @@
++			};
++
++			pmx_led_sys_green: pmx-led-sys-green {
++				marvell,pins = "mpp28";
++				marvell,function = "gpio";
++			};
++
++			pmx_led_sys_orange: pmx-led-sys-orange {
++				marvell,pins = "mpp29";
++				marvell,function = "gpio";
++			};
++
++			pmx_led_sata1_green: pmx-led-sata1-green {
++				marvell,pins = "mpp41";
++				marvell,function = "gpio";
++			};
++
++			pmx_led_sata1_red: pmx-led-sata1-red {
++				marvell,pins = "mpp42";
++				marvell,function = "gpio";
++			};
++
++			pmx_beeper: pmx-beeper {
++				marvell,pins = "mpp44";
++				marvell,function = "gpio";
++			};
++
++			pmx_pwr_sata1: pmx-pwr-sata1 {
++				marvell,pins = "mpp47";
++				marvell,function = "gpio";
++			};
++
++		};
++
++		/* This board uses the pcf8563 RTC instead of the SoC RTC */
++		rtc@10300 {
++			status = "disabled";
++		};
++
++		i2c@11000 {
++			status = "okay";
++
++			pcf8563: pcf8563@51 {
++				compatible = "nxp,pcf8563";
++				reg = <0x51>;
++			};
++		};
++
++	};
++
++	gpio-leds {
++		compatible = "gpio-leds";
++		pinctrl-0 = <&pmx_led_sata1_green &pmx_led_sata1_red
++			     &pmx_led_sata2_green &pmx_led_sata2_red
++			     &pmx_led_sys_green &pmx_led_sys_orange
++			     &pmx_led_copy_green &pmx_led_copy_red
++			     &pmx_led_usb_green>;
++		pinctrl-names = "default";
++
++		green-sata2 {
++			label = "nsa325:green:sata2";
++			gpios = <&gpio0 12 GPIO_ACTIVE_HIGH>;
++                        linux,default-trigger = "ide-disk2";
++		};
++
++		red-sata2 {
++			label = "nsa325:red:sata2";
++			gpios = <&gpio0 13 GPIO_ACTIVE_HIGH>;
++		};
++
++		green-usb {
++			label = "nsa325:green:usb";
++			gpios = <&gpio0 15 GPIO_ACTIVE_HIGH>;
++                        linux,default-trigger = "usb-host";
++		};
++
++		green-sys {
++			label = "nsa325:green:sys";
++			gpios = <&gpio0 28 GPIO_ACTIVE_HIGH>;
++			linux,default-trigger = "default-on";
++		};
++
++		orange-sys {
++			label = "nsa325:orange:sys";
++			gpios = <&gpio0 29 GPIO_ACTIVE_HIGH>;
++			linux,default-trigger = "cpu0";
++		};
++
++		green-copy {
++			label = "nsa325:green:copy";
++			gpios = <&gpio1 7 GPIO_ACTIVE_HIGH>;
++		};
++
++		red-copy {
++			label = "nsa325:red:copy";
++			gpios = <&gpio1 8 GPIO_ACTIVE_HIGH>;
++		};
++
++		green-sata1 {
++			label = "nsa325:green:sata1";
++			gpios = <&gpio1 9 GPIO_ACTIVE_HIGH>;
++			linux,default-trigger = "ide-disk1";
++		};
++
++		red-sata1 {
++			label = "nsa325:red:sata1";
++			gpios = <&gpio1 10 GPIO_ACTIVE_HIGH>;
++		};
++	};
++
++	regulators {
++		compatible = "simple-bus";
++		#address-cells = <1>;
++		#size-cells = <0>;
++		pinctrl-0 = <&pmx_usb_power_off &pmx_pwr_sata1>;
++		pinctrl-names = "default";
++
++		usb_power: regulator@1 {
++			compatible = "regulator-fixed";
++			reg = <1>;
++			regulator-name = "USB Power";
++			regulator-min-microvolt = <5000000>;
++			regulator-max-microvolt = <5000000>;
++			regulator-always-on;
++			regulator-boot-on;
++			enable-active-high;
++			gpio = <&gpio0 21 GPIO_ACTIVE_HIGH>;
++		};
++
++/************
```

Comment:
Due to the fact that this is a dtb supposed to be upstreamed, I don't think it's good to remove the comments here. If people want to stop the watchdog from the kernel this is how it is done.